### PR TITLE
New inputs for benchmark workflow

### PR DESCRIPTION
**Pull request by pruivo**
* CPU and Memory
* Custom Keycloak image
* Fixed Openshift project name

### CODE REVIEW

**Comment by pruivo** [comment]
Got a successful run in my private repo: https://github.com/pruivo/keycloak-benchmark/actions/runs/5143287976

File: .github/workflows/rosa-run-benchmark.yml [hunk 1]
```diff
@@ -5,14 +5,38 @@ on:
     inputs:
       clusterName:
         description: 'Name of the cluster'
-        type: text
+        type: string
+      namespacePrefix:
```
Comment (mhajas) [inline .github/workflows/rosa-run-benchmark.yml:9]: Do we have some use-case for setting `namespacePrefix` here? Thanks to 
```
concurrency: cluster_${{ inputs.clusterName || format('gh-{0}', github.repository_owner) }}
```
we are sure there is only one benchmark workflowrunning at each time. Is there some advantage of running each in a different namespace?

Comment (kami619) [reply .github/workflows/rosa-run-benchmark.yml:9]: Ideally we should avoid running multiple benchmarks against different namespaces in parallel.

File: .github/workflows/rosa-run-benchmark.yml [hunk 2]
```diff
@@ -5,14 +5,38 @@ on:
...
+      disableStickySessions:
+        description: 'Disable sticky session in Openshift Route'
+        type: boolean
+        default: false
+        required: true
+      image:
+        description: 'Keycloak Image'
+        type: string
+      podMemoryLimit:
+        description: 'Keycloak pod memory limit (in MB)'
```
Comment (kami619) [inline .github/workflows/rosa-run-benchmark.yml:25]: we would need to set some default values, if we are going to introduce limits, also consider adding requests, as requests and limits go hand in hand.

File: .github/workflows/rosa-run-benchmark.yml (hunk 1 above)
Comment (pruivo) [reply .github/workflows/rosa-run-benchmark.yml:9]: ok, I'll hardcode the prefix.

File: .github/workflows/rosa-run-benchmark.yml [hunk 3]
```diff
@@ -65,12 +92,19 @@ jobs:
         run: task
         working-directory: provision/openshift
         env:
-          KC_NAMESPACE_PREFIX: ''
+          KC_NAMESPACE_PREFIX: ${{ env.NAMESPACE_PREFIX }}
```
Comment (kami619) [inline .github/workflows/rosa-run-benchmark.yml:95]:
```suggestion
          KC_NAMESPACE_PREFIX: ${{ env.PROJECT_PREFIX }}
```

**Pull request merged**